swagger.yaml

```diff
@@ -0,0 +1,389 @@
+      $ref: '#/definitions/State'
+    required: true
+  nodeId:
+    name: nodeId
+    description: ""
+    in: body
+    schema:
+      $ref: '#/definitions/NodeId'
+    required: true
+  nodeName:
+    name: nodeName
+    description: ""
+    type: string
+    in: path
+    required: true
+securityDefinitions:
+  FlorenceAPIKey:
+    description: "API key used to allow florence users to create and query the progress of importing a dataset"
+    type: apiKey
+    in: header
+    name: florence-token
+  InternalAPIKey:
+    description: "API key used to allow only internal services to update the state of an import job"
+    type: apiKey
+    in: header
+    name: internal-token
+paths:
+  /job:
+    post:
+      tags:
+      - "External Interface"
+      summary: "Create an import job"
+      description: |
+        To generate a dataset a single or multiple files are required. Each file must be store in S3 and the full S3 path must be provided
+        in the body of the message. A name, edition and a version for the dataset must all be provided.
+      produces:
+      - "application/json"
+      parameters:
+      - $ref: '#/parameters/importJob'
+      security:
+      - FlorenceAPIKey: []
+      responses:
+        200:
+          description: "A import job was successfully created"
+          schema:
+            $ref: '#/definitions/Job'
+        400:
+          description: "Invalid json message was sent to the API"
+        500:
+          description: "Failed to process the request due to an internal error"
+  /job/{jobId}/s3file:
+    put:
+      tags:
+      - "External Interface"
+      summary: "Add a file to the import job"
+      description: "Add a file into the import job"
+      parameters:
+      - $ref: '#/parameters/jobId'
+      - $ref: '#/parameters/s3File'
+      produces:
+      - "application/json"
+      security:
+      - FlorenceAPIKey: []
+      responses:
+        200:
+          description: "The file was added to the import job"
+        400:
+          description: "Invalid json message was sent to the API"
+        404:
+          description: "InstanceId does not match any import jobs"
+        500:
+          description: "Failed to process the request due to an internal error"
+  /job/{jobId}/state:
+    put:
+      tags:
+      - "External Interface"
+      summary: "Start importing the dataset"
+      description: "Once this is called the import job is placed into a queue to be proccessed"
+      parameters:
+      - $ref: '#/parameters/jobId'
+      produces:
+      - "application/json"
+      security:
+      - FlorenceAPIKey: []
+      responses:
+        200:
+          description: "The import job is in a queue"
+        400:
+          description: "Invalid json message was sent to the API"
+        404:
+          description: "InstanceId does not match any import jobs"
+        500:
+          description: "Failed to process the request due to an internal error"
+  /instance/{instanceId}:
```

Comment:
Surely this description and summary are wrong or the request url is wrong, should it be either:

GET / instance/{instanceId}
description: "Get the current state of an instance of a particular job, this includes all events which have happened and an overall state"

OR

GET /job/{jobId}
description: "Get the current state of an import job, this includes all events which have happened and a overall state of the job."